src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -143,4 +152,15 @@ public static void showPersonAtIndex(Model model, Index targetIndex) {
         assertEquals(1, model.getFilteredPersonList().size());
     }
 
+    /**
+     * Updates {@code model}'s filtered list to show only the person's assignment at the given {@code targetIndex}
+     * in the {@code model}'s address book.
+     */
+    public static void showAssignmentAtIndex(Model model, Person person, Index targetIndex) {
+        //Assignment assignment = TypicalAssignments.getTypicalAssignments().get(targetIndex.getZeroBased());
+        // person.getAssignments().add(assignment);
```

Comment:
Same as first comment

Reply:
this is supposed to be modified when i submit the delete and mark assignment test, should be included now